pkg/rules/packages-to-build.yml

```diff
@@ -0,0 +1,52 @@
+# This matrix definition is used as both the package_build_rules and the
+# package_test_rules Ploutos packaging workflow inputs.
+---
+pkg:
+  - "dnsi"
+image:
+  - "ubuntu:xenial"   # ubuntu/16.04
+  - "ubuntu:bionic"   # ubuntu/18.04
+  - "ubuntu:focal"    # ubuntu/20.04
+  - "ubuntu:jammy"    # ubuntu/22.04
+  - "debian:stretch"  # debian/9
+  - "debian:buster"   # debian/10
+  - "debian:bullseye" # debian/11
+  - "debian:bookworm" # debian/12
+  - 'centos:7'
+  - 'rockylinux:8'    # compatible with EOL centos:8
+  - 'rockylinux:9'
+target:
+  - 'x86_64'
+test-image:
+  - ""
+include:
+  - pkg: "rtrtr"
+    image: 'debian:bullseye'
+    target: 'armv7-unknown-linux-musleabihf'
+
+  - pkg: 'dnsi'
+    image: 'debian:buster'
+    target: 'arm-unknown-linux-musleabihf'
+
+  - pkg: 'dnsi'
+    image: 'rockylinux:9'
+    target: 'x86_64'
+    test-image: 'almalinux:9'
+
+  - pkg: 'dnsi'
+    image: 'rockylinux:9'
+    target: 'x86_64'
+    test-image: 'centos:9-Stream'
+
+test-mode:   
+  - 'fresh-install'
+  - 'upgrade-from-published'
```

Comment:
upgrade-from-published should be removed as there is no prior version published to upgrade from.

Reply:
Ah, right. So I guess commenting it out is best.